classes/Search.php

```diff
@@ -229,22 +240,36 @@ public static function find(
             return false;
         }
 
+        $eligible_products2 = array();
```

Comment:
Prefer shortarray syntax :)
```suggestion
        $eligible_products2 = [];
```

Reply:
Done